templates/config-local.js

```diff
@@ -0,0 +1,54 @@
+/* Copyright (C) 2023 NooBaa */
```

Comment:
Few questions:

1. are those values documented
2. going to be changed in the future?
3. Where we got hose exactly?

Reply:
https://github.com/noobaa/noobaa-core/blob/master/docs/standalone.md#1-environment

Reply:
It might be worth to add this link as comment to this file.